src/system/packages/brew/shim.rb

```
@@ -0,0 +1,885 @@
+    entries = dest.children
+    return entries.first if entries.size == 1 && entries.first.directory?
+    dest
+  end
+
+  def system_or_die(*args)
+    raise "command failed: #{args.join(" ")}" unless system(*args)
+  end
+end
+
+module OS
+  def self.mac? = RbConfig::CONFIG["host_os"].include?("darwin")
```

RuboCop (1.87.0)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)